SPECS/golang/golang.spec

```diff
@@ -1,4 +1,6 @@
-%global bootstrap_compiler_version 1.19.12
+%global bootstrap_compiler_version_0 1.17.13
+%global bootstrap_compiler_version_1 1.21.6
+%global bootstrap_compiler_version_2 1.22.4
```

Comment:
I don't understand. It looks like 22.4 is the final version we're releasing with this spec, so why use it as a bootstrap step?